products/kms/api.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Kms
+display_name: Cloud KMS
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://cloudkms.googleapis.com/v1/
+    default: true
+scopes:
+  - https://www.googleapis.com/auth/cloudkms
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Cloud Key Management Service (KMS) API
+    url: https://console.cloud.google.com/apis/library/cloudkms.googleapis.com/
+objects:
+  - !ruby/object:Api::Resource
+    name: 'KeyRing'
+    base_url: 'projects/{{project}}/locations/{{location}}/keyRings?keyRingId={{name}}'
+    self_link: 'projects/{{project}}/locations/{{location}}/keyRings/{{name}}'
+    input: true
+    description: |
+      A `KeyRing` is a toplevel logical grouping of `CryptoKeys`.
+    collection_url_response: !ruby/object:Api::Resource::ResponseList
+      items: 'keyRings'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          The resource name for the KeyRing.
+        required: true
+      - !ruby/object:Api::Type::String
+        name: 'location'
+        description: |
+          The location for the KeyRing.
```

Comment:
The old docs said "A full list of valid locations can be found by running `gcloud kms locations list`", which I rather liked because otherwise I wouldn't necessarily realize 'global' is an option. Mind adding that (or something similar) back in?